runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -1223,11 +1223,11 @@ fn compute_entropy<T: Config>(parent_hash: T::Hash) -> [u8; 32] {
 ///   1. Check validity of all dispute statements sequentially
 /// 2. If not exceeded:
 ///   1. Sort the disputes based on locality and age, locality first.
-///   1. Split the array
-///   1. Prefer local ones over remote disputes
-///   1. If weight is exceeded by locals, pick the older ones (lower indices)
+///   2. Split the array
+///   3. Prefer local ones over remote disputes
+///   4. If weight is exceeded by locals, pick the older ones (lower indices)
 ///      until the weight limit is reached.
-///   1. If weight is exceeded by locals and remotes, pick remotes
+///   5. If weight is exceeded by locals and remotes, pick remotes
```

Comment:
```suggestion
///   4. If weight is exceeded by locals and remotes, pick remotes
```
this would follow the above numbering where the if has the same indices